lib/App/cpanminus/script.pm

```diff
@@ -91,6 +91,7 @@ sub new {
         installed_dists => 0,
         install_types => ['requires'],
         with_develop => 0,
+        with_configure => undef,
```

Comment:
Having pull request together and thinking this logic through again has made me spot a mistake in the logic for when installing configure dependencies when reading the dependencies from META files. Fix incomming...